tests/contrib/hooks/test_ftp_hook.py

```diff
@@ -96,6 +96,15 @@ def test_mod_time_micro(self):
 
         self.conn_mock.sendcmd.assert_called_once_with('MDTM ' + path)
 
+    def test_get_size(self):
+        self.conn_mock.get_size.return_value = '213 1942'
```

Comment:
Shouldn't you mock `size` instead of `get_size`?

Reply:
According to https://docs.python.org/2/library/ftplib.html#ftplib.FTP.size `size` returns `int`, not `str`.